examples/snowpeak/README.md

```diff
@@ -326,17 +326,17 @@ service /snowpeak on new http:Listener(port) {
    resource function get locations/[string id]/rooms(string startDate, string endDate)
                returns rep:Rooms|rep:SnowpeakInternalError {}
  
-   resource function put reservations(@http:Payload rep:Reservation reservation)
-               returns rep:ReservationCreated|rep:ReservationConflict|rep:SnowpeakInternalError {}   
+   resource function post reservations(@http:Payload rep:Reservation reservation)
+               returns rep:ReservationReceipt|rep:ReservationConflict|rep:SnowpeakInternalError {}   
              
    resource function put reservations/[string id](@http:Payload rep:Reservation reservation)
                returns rep:ReservationUpdated|rep:ReservationConflict|rep:SnowpeakInternalError {}
 
    resource function delete reservations/[string id]() returns
                rep:ReservationCanceled|rep:SnowpeakInternalError {}
  
-   resource function put payments/[string id](@http:Payload rep:Payment payment)
-               returns rep:PaymentCreated|rep:PaymentConflict|rep:SnowpeakInternalError {}
+   resource function post payments/[string id](@http:Payload rep:Payment payment)
+               returns rep:PaymentReceipt|rep:PaymentConflict|rep:SnowpeakInternalError {}
```

Comment:
Readme has more `ReservationCreated` usage. For example L384

Reply:
Fixed